tools/topology/topology1/development/sof-tgl-nocodec-ci.m4

```diff
@@ -122,7 +122,7 @@ PIPELINE_PCM_ADD(sof/pipe-volume-demux-playback.m4,
 
 # Passthrough capture pipeline 2 on PCM 0 using max 2 channels of s16le.
 # Schedule 48 frames per 1000us deadline on core 0 with priority 0
-PIPELINE_PCM_ADD(sof/pipe-passthrough-capture.m4,
+PIPELINE_PCM_ADD(sof/pipe-volume-switch-capture.m4,
```

Comment:
while you're at changing this, can you also change the comment just above to say volume-switch capture pipeline instead of passthrough?

Reply:
hi @ranj063  I modified the comments : passthrough -> volume-switch and s16->s32 as what PIPELINE_PCM_ADD claim below